### ISSUE TITLE
Windows 3.0a and QEMM 6.02 software

### ISSUE THREAD

**Issue by Fenix770** [86Box/86Box#1441]
**Describe the bug**

The VM runs fine, but there are issues in the real and enhanced mode of Windows 3.0, regarding the management of the EMS memory.
The machine is the NCR 386SX with DOS 5.0, Windows 3.0a and QEMM 6.02. This version of QEMM was launched in 1991.

1. When starting in real mode the machine just crashes when loading any application, if the EMS memory is present. An error message appears: 

![Win 3 0 real mode](https://user-images.githubusercontent.com/31600563/118546737-bcde5c80-b72e-11eb-8714-a05a262f3ceb.png)

2. In 386 enhanced mode Windows apps works fine, however the problem appears when I open a DOS session. An error window says the application has violated the integrity of the system

![Win 3 0 Enhanced mode](https://user-images.githubusercontent.com/31600563/118548459-b650e480-b730-11eb-92b7-85beaabf6ca2.png)

**To Reproduce**
1. Boot the machine
2. Try the real and enhanced mode of Windows to reproduce the bug.

Here is the NVR and HD images:

[386 SX  Cirrus Logic 5420 Win 3.0.zip](https://github.com/86Box/86Box/files/6496484/386.SX.Cirrus.Logic.5420.Win.3.0.zip)

[386 SX Cirrus Logic 5420 ESDI 723-13-51.zip](https://github.com/86Box/86Box/files/6496486/386.SX.Cirrus.Logic.5420.ESDI.723-13-51.zip)

**Expected behavior**
This seems to be a bug, also the readme file included in QEMM does not mention anything about Windows 3.0.

**Desktop (please complete the following information):**
 - OS:  Windows 10 ver 2004
 - 86Box version: 3.0.0.2988
 - Build type: Dev

**Additional context**
I installed a DOS chess game that uses EMS memory for the hash tables. It can use from 512kb to 8mb of LIM EMS memory on 80386 CPUs. I played some moves and seems to work just fine. (It's installed on C:\CCSYSTEM directory)

**Comment by TC1995** [comment]
I don't think the 5420 has high/true color capabilities.

**Comment by Fenix770** [comment]
I think this issue may be related to either a bug in CPU emulation or the way QEMM emulates EMS memory rather the video chip.

**Comment by Fenix770** [comment]
I discovered recently there is special switches in the QEMM386.SYS. I installed QEMM 5.0 and them v6. 02, then when an older version is detected  the installer offers various options to the user.

1.I Created the EMS page frame at D000-DFFF region, which is more common and
2.The amount alternate page registers in 16 instead of 8 and
3. The option to copy BIOS data from ROM to RAM.

Solved problems:

a. The real mode bug  has gone
b. The error no longer appears when attempting to open a DOS window in 386 enhanced mode

The only issue which remains is:

1. The mouse disapperas when exiting from DOS prompt in 386 enhanced mode
2. Less upper memory available.

Here is the new copy of the hdd:
[386 SX  Cirrus Logic 5420 ESDI 723-13-51.zip](https://github.com/86Box/86Box/files/6504818/386.SX.Cirrus.Logic.5420.ESDI.723-13-51.zip)

Screenshots:

![QEMM-1](https://user-images.githubusercontent.com/31600563/118733621-fe483800-b812-11eb-8d4c-9c12ba843ecb.png)

![QEMM-2](https://user-images.githubusercontent.com/31600563/118733623-0011fb80-b813-11eb-9885-24d8f720ae0d.png)
